src/developer/functions/https/keys.js

```diff
@@ -39,25 +39,31 @@ export const getKeys = async (uid) => {
   const keys = [];
   querySnapshot.forEach((doc) => keys.push(doc.data()));
 
-  const usageKeys = keys.map((key) => `analytics:hit:${key.key.toLowerCase()}`);
-  const timeKeys = keys.map((key) => `analytics:time:${key.key.toLowerCase()}`);
-
-  const allKeys = [...usageKeys, ...timeKeys];
-
-  debug('getting usage information for', allKeys);
-
-  try {
-    let results = await redis.mget(...allKeys);
-
-    results.forEach((result, index) => {
-      if (index < keys.length) {
-        keys[index].usage = result ?? 0;
-      } else {
-        keys[index - keys.length].lastUsed = result ?? -1;
-      }
-    });
-  } catch (ex) {
-    error('redis error', ex);
+  if (skipAnalytics) {
```

Comment:
```suggestion
  if (!skipAnalytics) {
```
Should this be flipped?

Reply:
no, but the variable should be renamed...